pom.xml

```diff
@@ -196,6 +197,10 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+        <repository>
```

Comment:
Why you doesn't want T4C repo into the main POM? It's quite anoying to import the repository everytime I want to import T4C  modules.